packages/core/src/components/forms/_input-group.scss

```diff
@@ -144,53 +160,53 @@ $input-button-height-large: $pt-button-height !default;
       line-height: 0;
     }
 
-    .pt-icon {
+    > .pt-icon-standard {
       margin: 0 ($pt-input-height-large - $pt-icon-size-standard) / 2;
-      line-height: $pt-input-height-large;
     }
 
     .pt-input {
       @include pt-input-large();
-
-      &:not(:first-child) {
-        padding-left: $pt-button-height-large;
-      }
-
-      &:not(:last-child) {
-        padding-right: $pt-button-height-large;
-      }
     }
 
     .pt-input-action > .pt-spinner {
       margin: ($pt-input-height-large - $spinner-width * $spinner-width-factor-small) / 2;
     }
   }
 
-  &.pt-round {
-    .pt-button,
-    .pt-input,
-    .pt-tag {
-      border-radius: $pt-input-height;
-    }
+  &.pt-round,
+  &.pt-round .pt-button,
+  &.pt-round .pt-tag, {
+    border-radius: $pt-input-height;
```

Comment:
why make this change? the previous syntax was shorter and more obvious.

Reply:
@giladgray mmm, I'm not sure haha. Will revert.

Reply:
@giladgray stashing this and punting for now